homeassistant/components/kef/media_player.py

```diff
@@ -0,0 +1,270 @@
+"""Platform for the KEF Wireless Speakers."""
+
+import datetime
```

Comment:
Just a small thing ✌
Since only the used module is imported in a lot of other integrations I would change this to
```python
from datetime import timedelta
```

Reply:
Done! Thanks :)

Reply:
No problem at all :)